OpenRA.Mods.Common/Widgets/Logic/MultiplayerLogic.cs

```diff
@@ -627,5 +627,11 @@ bool Filtered(GameServer game)
 
 			return false;
 		}
+
+		void CloseWindow()
```

Comment:
Overriding `ChromeLogic.Dispose` would be more robust, as that is called automatically no matter how the window is closed.